jekyll/_cci2/runner-faqs-adoc

```diff
@@ -0,0 +1,58 @@
+= CircleCI Runner FAQs
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== What is the pricing for the CircleCI runner?
+
+Currently, CircleCI does not charge for runner usage; however, this will change in the future. If you are a "preview" customer, your account team will keep you informed of any pricing changes for the runner.
+
+=== What is the security model for the runner?
+
+When installing the runner you will be able to choose the user that executes jobs. It is up to you to ensure this user only has permissions you are comfortable letting jobs use. 
+
+NOTE: Allowing jobs to access a docker daemon is equivalent to providing root access to the machine.
+
+=== How do I install dependencies needed for my jobs?
+
+There are two main approaches available for installing dependencies:
+
+==== Allow the jobs to install their own dependencies
+
+This approach is the most flexible, but will require providing the jobs sufficient privileges to install tools or install the tools in a non-overlapping manner (eg. into the working directory).
+
+==== Pre-install dependencies on the runner machine
+
+This approach is the most secure; however, this means that if the job’s dependencies change, the runner machine must be reconfigured.
+
+=== What connectivity is required?
+
+In order to connect back to CircleCI to receive and execute jobs, outbound HTTPS connections to `runner.circleci.com`, `circleci-binary-releases.s3.amazonaws.com` are required.
+
+NOTE: No inbound connectivity is required by runner. Any other required connectivity is dependent on the content of the jobs themselves.
+
+CAUTION: Using the checkout step will require access to your VCS provider. Using the cache, workspace or artifact features will require outbound HTTPS connections to `circle-production-customer-artifacts.s3.amazonaws.com`.
+
+=== How do caching and workspaces and artifacts work with the runner?
+
+Caches and workspaces and artifacts will be stored in the us-east-1 region of S3. If your runners are not in this region then you may see reduced performance. Although we are not currently charging for runner usage, the transfer and storage of data will incur costs when pricing is introduced.
```

Comment:
Let’s rephrase the charging part as “we expect to charge for data transfer and storage in the future”.

Reply:
Fixed